scribble-lib/scribble/acmart/lang.rkt

```diff
@@ -147,3 +156,18 @@
   (struct-copy part doc [to-collect
                          (cons (terms) ; FIXME
                                (part-to-collect doc))]))
+
+(define (acmart.cls)
```

Comment:
I'm thinking about moving this to a helper function with type `string? (-> path-string? void?) -> void?` and sharing between `scribble/jfp`.